app/models/phone.rb

```diff
@@ -1,8 +1,3 @@
 class Phone < Channel
-  attr_accessible :carrier, :priority, :sms_available
 
-  belongs_to :person
-
-  PRIORITIES = ['1-Call First','2','3','4-Call Last']
-  CARRIERS = ['Sprint','Verizon','AT&T','Metro PCS','T-Mobile','Comcast','Cricket', 'Other']
```

Comment:
These are referenced in the views to have one source for these choices. 
Why delete ?